services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/LinkPropertiesSpec.groovy

```diff
@@ -164,6 +164,18 @@ class LinkPropertiesSpec extends BaseSpecification {
         islUtils.getIslInfo(updatedLinks, isl).get().maxBandwidth == maxBandwidthValue.toInteger()
         islUtils.getIslInfo(updatedLinks, isl.reversed).get().maxBandwidth == maxBandwidthValue.toInteger()
 
+        when: "Update cost on forward direction of the link props"
```

Comment:
Sorry, don't understand the sense of newly added code block. It is already verified at L164 and L165.

Reply:
in mentioned lines we create a new linkProps for an isl
here we update one direction of the created linkProps

Reply:
1. Ok, got it. So can we improve description at L152 then? For example, `Create link props on the forward direction of ISL to update cost and max bandwidth`.
2. And here `Update link props on the forward direction of ISL to update cost one more time`. 
3. Also, please add some check after L156 that count of created link props is 2.

Reply:
1. changed to `"Create link props of ISL to update cost and max bandwidth in the forward and reverse directions"`
2. changed to your proposal
3. added